components/Pagination/src/stories/pagination.bem.stories.mdx

```diff
@@ -0,0 +1,678 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+
+import "../index";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="CSS Components/Navigation/Pagination"
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    docs: { source: { type: "dynamic" } },
+    status: {
+      type: "IN DEVELOPMENT",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="Pagination" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <nav class="denhaag-pagination">
+      <a href="#page-1" class="denhaag-pagination__link denhaag-pagination__link--arrow" rel="prev">
+        <svg
+          class="denhaag-icon"
+          width="1em"
+          height="1em"
+          viewBox="0 0 7 12"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
```

Comment:
The next and previous arrow SVGs should have an `aria-hidden="true"` attribute. Otherwise, screen readers will announce a meaningless _image_